packages/service-library/tests/fastapi/test_httpx_utils.py

```diff
@@ -61,7 +62,7 @@ async def test_to_curl_command(client: AsyncClient):
 
     assert (
         cmd_short
-        == 'curl -X POST -H "host: test_base_http_api" -H "accept: */*" -H "accept-encoding: gzip, deflate" -H "connection: keep-alive" -H "user-agent: python-httpx/0.25.0" -H "x-secret: *****" -H "content-length: 9" -H "content-type: application/json" -d \'{"y": 12}\' https://test_base_http_api/foo?x=3'
+        == f'curl -X POST -H "host: test_base_http_api" -H "accept: */*" -H "accept-encoding: gzip, deflate" -H "connection: keep-alive" -H "user-agent: python-httpx/0.25.0" -H "x-secret: {_PLACEHOLDER}" -H "content-length: 9" -H "content-type: application/json" -d \'{{"y": 12}}\' https://test_base_http_api/foo?x=3'
```

Comment:
Q: is this change correct? I would have expected for the secret to be naked like it was before.

Reply:
what do you mean by naked?  This is as it was before but instead of hard-coding the `***`s I am using the placeholder